src/makielayout/lineaxis.jl

```diff
@@ -581,18 +581,20 @@ function get_ticks(tickfunction::Function, any_scale, formatter, vmin, vmax)
     return tickvalues, ticklabels
 end
 
+_logbase(s::ReversibleScale) = s.logbase
 _logbase(::typeof(log10)) = "10"
 _logbase(::typeof(log2)) = "2"
 _logbase(::typeof(log)) = "e"
+_logbase(::Any) = ""
```

Comment:
this should probably be `nothing`, `get_ticks` will be specialized anyway

Reply:
Right, this was in order to simplify the `ReversibleScale` `logbase` type which was a `String` and now has to be a `Union{Nothing,String}`.